‘fail’ is not a (visible) method of class ‘Monad’

From latest commit, ghc ` 8.8.3`, base `4.13.0.0`:
```shell
transformers-abort               > /run/user/1001/stack-62dbeefc3899fae0/transformers-abort-0.6.0.1/src/Control/Monad/Trans/Abort.hs:113:3: error:
transformers-abort               >     ‘fail’ is not a (visible) method of class ‘Monad’
transformers-abort               >     |              
transformers-abort               > 113 |   fail    = AbortT . fail
transformers-abort               >     |   ^^^^       
transformers-abort               >                    
```

--- a/src/Control/Monad/Trans/Abort.hs
+++ b/src/Control/Monad/Trans/Abort.hs
@@ -110,9 +110,11 @@ instance (Functor μ, Monad μ) ⇒ Bind (AbortT e μ) where
 instance Monad μ ⇒ Monad (AbortT e μ) where
   return  = AbortT . return . Right
   m >>= f = AbortT $ runAbortT m >>= either (return . Left) (runAbortT . f)
+#if !MIN_VERSION_base(4,13,0)
   fail    = AbortT . fail
-
+#endif
 #if MIN_VERSION_base(4,9,0)
+
 instance Fail.MonadFail μ ⇒ Fail.MonadFail (AbortT f μ) where
   fail = AbortT . Fail.fail
 #endif

--- a/src/Control/Monad/Trans/Finish.hs
+++ b/src/Control/Monad/Trans/Finish.hs
@@ -107,9 +107,11 @@ instance (Functor μ, Monad μ) ⇒ Bind (FinishT f μ) where
 instance Monad μ ⇒ Monad (FinishT f μ) where
   return  = FinishT . return . Right
   m >>= f = FinishT $ runFinishT m >>= either (return . Left) (runFinishT . f)
+#if !MIN_VERSION_base(4,13,0)
   fail    = FinishT . fail
-
+#endif
 #if MIN_VERSION_base(4,9,0)
+
 instance Fail.MonadFail μ ⇒ Fail.MonadFail (FinishT f μ) where
   fail = FinishT . Fail.fail
 #endif

--- a/transformers-abort.cabal
+++ b/transformers-abort.cabal
@@ -1,5 +1,5 @@
 Name: transformers-abort
-Version: 0.6.0.1
+Version: 0.6.0.2
 Category: Control
 Stability: experimental
 Synopsis: Error and short-circuit monad transformers

--- a/transformers-abort.cabal
+++ b/transformers-abort.cabal
@@ -19,7 +19,7 @@ Extra-Source-Files:
   README.md
 
 Tested-With: GHC==7.6.3, GHC==7.8.4, GHC==7.10.3, GHC==8.0.2, GHC==8.2.2,
-             GHC==8.4.4, GHC==8.6.1
+             GHC==8.4.4, GHC==8.6.1, GHC==8.8.3
 
 Cabal-Version: >= 1.6.0
 Build-Type: Simple

